test/system/trader.js

```diff
@@ -71,37 +59,43 @@ class Trader {
     }
   }
 
+  // execute buy-2/sell-2 orders for trader1
+  runScenario1() {
```

Comment:
The Trader class should try to offer a small set of reusable operations like 'placeOrderSpread', 'cancelAllOrders' and so on. The Scenario logic will typically need to control more than one trader and so is better off external to the Trader class